test/test_helper.exs

```diff
@@ -0,0 +1,86 @@
+defmodule SSEAssertions do
+  @stream_headers [{"content-type", "text/event-stream"}]
+
+  import ExUnit.Assertions, only: [assert: 1, assert: 2]
```

Comment:
That imports `Assertions.assert/1` and `Assertions.assert/2` instead of all functions in the module. We only use `assert/1`, but I'm guessing that calls `assert/2` internally or something.